tools-local/Microsoft.ML.InternalCodeAnalyzer/BaseTestClassAnalyzer.cs

```diff
@@ -59,7 +59,7 @@ public AnalyzerImpl(Compilation compilation, INamedTypeSymbol factAttribute)
                 _compilation = compilation;
                 _factAttribute = factAttribute;
                 _baseTestClass = _compilation.GetTypeByMetadataName("Microsoft.ML.TestFramework.BaseTestClass");
-                _FTbaseTestClass = _compilation.GetTypeByMetadataName("Microsoft.ML.Functional.Tests.FunctionalTestBaseClass");
+                _FTbaseTestClass = _compilation.GetTypeByMetadataName("Microsoft.ML.IntegrationTests.FunctionalTestBaseClass");
```

Comment:
nit: Can you please rename the class and variables to match the rename? (IntegrationTestBaseClass, etc.)